server/src/clj_kondo/lsp_server/impl/server.clj

```diff
@@ -114,14 +116,25 @@
                 (.getParentFile))]
     (config-dir dir)))
 
+(defn config-options
+  [arg ^java.io.File config-dir]
+  (let [file (io/file arg)]
+       (if (.isFile file)
+         (if (.isRelative file)
```

Comment:
Have you tested this locally? AFAIK Java doesn't have a `isRelative` method. It does have a `isAbsolute` method.